### PR TITLE
Hva er instrastruktur som kode?

### PR DESCRIPTION
**Pull request by simenandre**
Se artikkelen her:
https://bjerk-io--pr648-simenandre-article-h-2nyg8nxl.web.app/articles/2023/hva-er-infrastruktur-som-kode

### CODE REVIEW

File: src/content/post/2023/hva-er-infrastruktur-som-kode/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  distribusjon
+author:
+  - simenandre
+---
+
+- Hva? Hvorfor? Hvordan? Hvor?
+
+Det korte svaret er at _Infrastructure as Code_ eller IAC er en måte å beskrive,
+styre og provisjonere infrastruktur på. Det vil si, alt fra servere, nettverk,
+databaser og andre ting, og i dag er det kanskje mer vanlig at man provisjonerer
```
Comment (braaar) [inline src/content/post/2023/hva-er-infrastruktur-som-kode/index.md:20]:
```suggestion
databaser og andre ting, og i dag er det kanskje vanligst at man provisjonerer
```

File: src/content/post/2023/hva-er-infrastruktur-som-kode/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+disse tingene i en av de store skyleverandørene som Google Cloud, Azure og
+Amazon Web Services (AWS).
+
+Infrastruktur som kode har endret stort hvordan IT-ressurser håndteres og
```
Comment (braaar) [inline src/content/post/2023/hva-er-infrastruktur-som-kode/index.md:24]:
```suggestion
Infrastruktur som kode har endret hvordan IT-ressurser håndteres og
```

I feel like this sentence needs to be reworded if you want to add a strengthening adverb

File: src/content/post/2023/hva-er-infrastruktur-som-kode/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+styres. Med infrastruktur som kode kan du bruke prinsipper fra programmering,
+slik som versjoneringskontroll (Git), samarbeid og kontinuerlig integrasjon og
+levering (CI/CD) slik at organisasjoner kan bli mer effektive og automatisere
+prosesser.
+
+Verktøyene reduserer risiko for menneskelig feil, skaper hurtige og sikre
+deployments (eller utrulling). Måten det ofte fungerer på er at man definererer
```
Comment (braaar) [inline src/content/post/2023/hva-er-infrastruktur-som-kode/index.md:31]:
```suggestion
Verktøyene reduserer risiko for menneskelig feil, og skaper hurtige og sikre
deployments (utrullinger). Måten det ofte fungerer på er at man definerer
```

File: src/content/post/2023/hva-er-infrastruktur-som-kode/index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+en ønsket tilstand til infrastrukturen, eksempelvis at en _Deployment_ i
+Kubernetes skal kjøre en Docker-fil med en spesifisert tag, og så sørger
+verktøyet for at det blir sånn. Dette skiftet har også vært med på å skape en ny
+rolle i IT-organisasjoner, nemlig _DevOps_ (en slags sammensmeltet rolle mellom
+utvikling og IT-driftspersonell).
```
Comment (braaar) [inline src/content/post/2023/hva-er-infrastruktur-som-kode/index.md:36]:
```suggestion
utvikler og IT-driftspersonell).
```

File: src/content/post/2023/hva-er-infrastruktur-som-kode/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+## Verktøyene
+
+Det finnes mange verktøy, det første verktøyet som tok verden med storm var nok
+Chef, som ble lansert i 2009. Siden den gang har det kommet mange verktøy, slik
+som Puppet, Terraform, Ansible, Pulumi, CloudFormation, CDK, Serverless og Helm.
```
Comment (braaar) [inline src/content/post/2023/hva-er-infrastruktur-som-kode/index.md:42]:
```suggestion
Det finnes mange verktøy. Det første verktøyet som tok verden med storm var nok
Chef, som ble lansert i 2009. Siden den gang har det kommet mange verktøy, slik
som Puppet, Terraform, Ansible, Pulumi, CloudFormation, CDK, Serverless og Helm.
```

File: src/content/post/2023/hva-er-infrastruktur-som-kode/index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  ami: 'ami-0c55b159cbfafe1f0',
+  instanceType: 't2.micro',
+  tags: {
+    Name: 'ExampleInstance',
+  },
+});
+```
+
+En av fordelene med Pulumi er at det er i et hvilket som helst
+programmeringsspråk. Alle verktøyene er basert på åpen kildekode, og personlig
```
Comment (braaar) [inline src/content/post/2023/hva-er-infrastruktur-som-kode/index.md:81]:
```suggestion
En av fordelene med Pulumi er at det lar deg bruke et hvilket som helst
programmeringsspråk. Alle verktøyene er basert på åpen kildekode, og personlig
```

File: src/content/post/2023/hva-er-infrastruktur-som-kode/index.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+har jeg bidratt både til Terraform og Pulumi.
+
+## Hva hjelper det meg med?
+
+Jeg tror det gir mening for de fleste å bruke infastruktur som kode på et eller
+annet tidspunkt når man lager et digitalt prod
```
Comment (braaar) [inline src/content/post/2023/hva-er-infrastruktur-som-kode/index.md:87]:
```suggestion
Jeg tror det gir mening for de fleste å bruke infastruktur som kode på et eller
annet tidspunkt når man lager et digitalt produkt.
```

**Review by braaar — COMMENTED** [review]
I made some suggestions to language improvements. Some of these might change the tone of the article somewhat, but they improve the flow of the text, in my opinion.

File: src/content/post/2023/hva-er-infrastruktur-som-kode/index.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+nemlig [_DevOps_][devops] (en slags sammensmeltet rolle mellom utvikler og
+IT-driftspersonell).
+
+[gcloud]: https://cloud.google.com/
+[azure]: https://azure.microsoft.com/
+[aws]: https://aws.amazon.com/
+[k8s]: https://kubernetes.io/
+[devops]: https://en.wikipedia.org/wiki/DevOps
+
+## Verktøyene
```
Comment (copetterson) [inline src/content/post/2023/hva-er-infrastruktur-som-kode/index.md:42]: Edit to: "Verktøyene for infrastruktur som kode"

**Pull request merged**